retirement/tasks.py

```diff
@@ -0,0 +1,31 @@
+from celery import shared_task
+from django.db import transaction
+from django.utils import timezone
+
+
+@shared_task
+def assign_retreat_tomatoes():
+    """
+    Assign tomatoes to all active users of a retreat if one of the date passed.
+    """
+    from retirement.models import RetreatDate
+    today = timezone.now()
+    to_assign_dates = RetreatDate.objects.filter(
+        end_time__lte=today,
+        tomatoes_assigned=False,
+    )
+
+    for date in to_assign_dates:
+        with transaction.atomic():
+            active_reservations = date.retreat.reservations.filter(
+                is_active=True)
```

Comment:
since it's possible that the member cancel the reservation AFTER the date and BEFORE the script run, it could be nice to base the filter on the `cancellation_date` of the `Reservation` object in place of just checking for `is_active`